kubernetes/resource_kubernetes_daemon_set_v1_test.go

```diff
@@ -912,6 +916,7 @@ func testAccKubernetesDaemonSetV1ConfigWithContainerSecurityContextSeccompProfil
       }
     }
   }
+  wait_for_rollout = false
```

Comment:
note: rollout of this resource fails on K8s nodes that do not have any seccomp profiles locally available (e.g. nodes in a `kind` cluster), so let's not wait for rollout, as it's not important for this test.